_posts/ja/newsletters/2024-12-06-newsletter.md

```diff
@@ -0,0 +1,169 @@
+  どの解決策を使用するべきかについてフィードバックを求めました（
+  この複製が行われると資金の喪失と攻撃ベクトルの作成につながる可能性があります）。
+  提案された解決策は4つありますが、いずれもマイナーに直接影響するものであるため、
+  [マイナーからのフィードバック][mining-dev]が特に望まれます。
+
+## リリースとリリース候補
+
+_人気のBitcoinインフラストラクチャプロジェクトの新しいリリースとリリース候補。
+新しいリリースにアップグレードしたり、リリース候補のテストを支援することを検討してください。_
+
+- [Eclair v0.11.0][]は、この人気のLNノード実装の最新リリースです。
+  「BOLT12 [オファー][topic offers]の公式サポートを追加し、
+  流動性管理機能（[スプライシング][topic splicing]、[Liquidity Ads][topic liquidity advertisements]、
+  [オンザフライ・ファンディング][topic jit channels]）が進捗しています」。
+  このリリースでは、新しく非[アンカーチャネル][topic anchor outputs]の受け入れも停止しています。
+  また、最適化やバグ修正も含まれています。
+
+- [LDK v0.0.125][]は、LN対応アプリケーションを構築するためのこのライブラリの最新リリースです。
+  いくつかのバグ修正が含まれています。
+
+- [Core Lightning 24.11rc3][]は、この人気のLN実装の次期メジャーバージョンのリリース候補です。
+
+- [LND 0.18.4-beta.rc1][]は、この人気のLN実装のマイナーバージョンのリリース候補です。
+
+- [Bitcoin Core 28.1RC1][]は、主要アンフルノード実装のメンテナンスバージョンのリリース候補です。
+
+## 注目すべきコードとドキュメントの変更
+
+_最近の[Bitcoin Core][bitcoin core repo]、[Core
+Lightning][core lightning repo]、[Eclair][eclair repo]、[LDK][ldk repo]、
+[LND][lnd repo]、[libsecp256k1][libsecp256k1 repo]、[Hardware Wallet
+Interface (HWI)][hwi repo]、[Rust Bitcoin][rust bitcoin repo]、[BTCPay
+Server][btcpay server repo]、[BDK][bdk repo]、[Bitcoin Improvement
+Proposals（BIP）][bips repo]、[Lightning BOLTs][bolts repo]、
+[Bitcoin Inquisition][bitcoin inquisition repo]および[BINANAs][binana repo]の注目すべき変更点。_
+
+- [Bitcoin Core #30708][]では、指定されたブロックハッシュのセット内の[ディスクリプター][topic descriptors]に関連付けられた
+  すべてのトランザクションを取得する`getdescriptoractivity` RPCコマンドが追加され、
+  ウォレットがステートレスな方法でBitcoin Coreとやりとりできるようになります。
+  このコマンドは、ディスクリプターに関連付けられたトランザクションを含むブロックハッシュのセットを識別する
+  `scanblocks`（[ニュースレター #222][news222 scanblocks]参照）と組み合わせて使用すると特に便利です。
+
+- [Core Lightning #7832][]では、緊急ではない一方的なクローズトランザクションの場合でも、
+  [アンカーアウトプット][topic anchor outputs]からの支払いをします。
+  ブロックターゲットは2016ブロック（約2週間）から徐々に12ブロックに縮小されます。
+  ブロードキャストした際のタイムスタンプは、再起動しても一貫した動作を保証するために追跡されます。
+  これまでは、これらのトランザクションはデフォルトでアンカーアウトプットから支払いをしていなかったため、
+  手動でその支払いを追加するのが困難で、[CPFP][topic cpfp]による手数料の引き上げが使用できませんでした。
+
+- [LND #8270][]は、[BOLT2][]で定義されているチャネル静止プロトコル（
+  ニュースレター[#309][news309 quiescence]参照）を実装します。これは、
+  [ダイナミックコミットメント][topic channel commitment upgrades]と[スプライシング][topic splicing]の前提条件です。
+  このプロトコルにより、ノードはピアの静止要求に応答し、新しい`ChannelUpdateHandler`操作を使用してプロセスを開始できます。
+  このPRでは、静止状態が解決されずに長時間続く場合に、ピアを切断して処理するための設定可能なタイムアウトの仕組みも追加されています。
+
+- [LND #8390][]は、[チャネルジャミング攻撃][topic channel jamming attacks]防止の研究を目的に、
+  `update_add_htlc`メッセージ内に実験的な[HTLCエンドースメント][topic htlc endorsement]の
+  シグナリングフィールドの設定とリレーのサポートを導入します。ノードがシグナリングフィールドを含むHTLCを受け取ると、
+  フィールドをそのままリレーします。それ以外の場合は、デフォルト値の0を設定します。
+  この機能はデフォルトで有効になっていますが、無効にすることもできます。
+
+- [BIPs #1534][]は、Taprootの内部鍵をスタックに配置する新しい[Tapscript][topic tapscript]専用のopcodeである
+  `OP_INTERNALKEY`の仕様を定義した[BIP349][]をマージしました。スクリプトの作成者は、
+  アウトプットに支払う前に内部鍵を知る必要があり、これは代わりにスクリプト内に直接内部鍵を含めるものです。
+  使用のたびに8 vbyteを節約し、スクリプトの再利用性を高めます（[ニュースレター #285][news285 bip349]参照）。
+
+{% include references.md %}
```

Comment:
@azuchi we added the recap call out back here

Reply:
@bitschmidty oh, thanks! I added.